From 2e0b8f27f566b8b1d13df088579ded78b6570be8 Mon Sep 17 00:00:00 2001
From: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Date: Wed, 10 Feb 2016 16:04:01 -0600
Subject: [PATCH] fix conflicted shebang path with FHS

This line causes a conflict for building images
in systems that use filesystem hierarchy standard (FHS)
because echo couldn't be provided.

Signed-off-by: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
---
 scripts/load-devel-config.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/load-devel-config.sh b/scripts/load-devel-config.sh
index 489f007..3c5e3dd 100755
--- a/scripts/load-devel-config.sh
+++ b/scripts/load-devel-config.sh
@@ -1,4 +1,4 @@
-#! /bin/echo must be loaded with .
+#!/usr/bin/echo must be loaded with .
 
 # Load DPDK devel config and allow override
 # from system file
-- 
2.5.0

